pkg/engine/engine.go

```diff
@@ -69,26 +68,28 @@ func FuncMap() template.FuncMap {
 	delete(f, "env")
 	delete(f, "expandenv")
 
-	// Add a function to convert to YAML:
-	f["toYaml"] = toYaml
+	// Add some extra functionality
+	extra := template.FuncMap{
+		"toYaml": chartutil.ToYaml,
+		"base":   path.Base,
```

Comment:
I guess my preference would be to have these path functions rolled into Sprig. We try to only put "special" functions here. If you put them in Sprig, I'll roll another Sprig release ASAP, too.